tests/migration/migration.go

```diff
@@ -766,37 +755,14 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 					WaitFor(ctx, watcher.WarningEvent, "migrationTargetPodUnschedulable")
 
 				By("Migration should observe a timeout period before canceling unschedulable target pod")
-				Consistently(func() error {
-
-					migration, err := virtClient.VirtualMachineInstanceMigration(migration.Namespace).Get(context.Background(), migration.Name, metav1.GetOptions{})
-					if err != nil {
-						return err
-					}
-
-					if migration.Status.Phase == v1.MigrationFailed {
-						return fmt.Errorf("Migration should observe timeout period before transitioning to failed state")
-					}
-					return nil
-
-				}, 1*time.Minute, 10*time.Second).Should(Succeed())
+				Consistently(ThisMigration(migration)).WithPolling(10 * time.Second).WithTimeout(1 * time.Minute).Should(Not(BeInPhase(v1.MigrationFailed)))
```

Comment:
Let's use the error constant as additional description...

Reply:
I think the `BeInPhase` already provides a quite good description, no?